### DISCUSSION TITLE
How to tell oras 1.0.0 not to push as oci-layout?

### DISCUSSION THREAD

**Discussion by jjcaballero** [oras-project/oras#947] [Q&A]
I just realized that I have been pulling artifacts with OCI-layout, with "layers" in manifest instead of "blobs", as I was doing until now.
I learned now from the release notes that this was caused because I was using oras 1.0.0 My bad.

A couple of questions:
- How to tell oras 1.0.0 not to push as oci-layout?
- Is there a command to change the manifest of an existing artifact in the registry?

Thanks!

**Comment by qweeah** [comment]
Are you suggesting that you have used `oras push` or `oras attach` to generate some artifacts with manifests typed as `application/vnd.oci.artifact.manifest.v1+json`, and you wish the type to be `application/vnd.oci.image.manifest.v1+json` instead？

If so, you may use `--image-spec v1.1-image` for your `push` or `attach` command.

And there is no command to change the manifest of existing artifact, you may repush and delete the artifacts with wrong types instead.
